static/js/compose.js

```diff
@@ -713,7 +713,28 @@ function get_or_set(fieldname, keep_leading_whitespace) {
     };
 }
 
-exports.stream_name     = get_or_set('stream');
+function get_contents(fieldname, keep_leading_whitespace) {
+
+    return function () {
```

Comment:
I think we need to allow for leading whitespace in the message_content field, but I would just handle that in message_content_get(), rather than passing the boolean around.